node/full.go

```diff
@@ -281,7 +281,13 @@ func (n *FullNode) Run(parentCtx context.Context) error {
 	}
 
 	// Start RPC server
-	handler, err := rpcserver.NewServiceHandler(n.Store, n.p2pClient, n.genesis.ProposerAddress, n.Logger, n.nodeConfig)
+	bestKnownHeightProvider := func() uint64 {
+		hHeight := n.hSyncService.Store().Height()
```

Comment:
the idea here is to push the check if the node is at the highest height onto the operator correct?